site/shared/connectors/metaMask.md

```diff
@@ -8,6 +8,8 @@ const connectorsPackageName = 'wagmi/connectors'
 
 Connector for [MetaMask SDK](https://github.com/MetaMask/metamask-sdk).
 
+The MetaMask Connector helps you manage wallet connections in your React applications. It supports essential features like connecting, disconnecting, and handling multiple chains, while maintaining compatibility with browser and mobile environments. You can choose between the MetaMask Connector and the Injected Connector—but keep in mind that using the Injected Connector limits your app to the MetaMask browser extension only.
```

Comment:
Since this is a shared page for the VanillaJS, Vue, and React docs, let's keep it basic and remove the mention of React.

```suggestion
Check out the [MetaMask SDK docs](https://docs.metamask.io/wallet/connect/metamask-sdk/javascript) for more information.
```